engine/table/src/main/java/io/deephaven/engine/table/impl/by/ChunkedOperatorAggregationHelper.java

```diff
@@ -86,34 +86,14 @@ public static QueryTable aggregation(
             final boolean preserveEmpty,
             @Nullable final Table initialKeys,
             @NotNull final Collection<? extends ColumnName> groupByColumns) {
-        final String[] keyNames = groupByColumns.stream().map(ColumnName::name).toArray(String[]::new);
-        if (!input.hasColumns(keyNames)) {
-            final Set<String> colNames = input.getColumnSourceMap().keySet();
-            final String[] missingColumns = Arrays.stream(keyNames)
-                    .filter(Predicate.not(colNames::contains))
-                    .toArray(String[]::new);;
-
-            throw new IllegalArgumentException("aggregation: not all group-by columns " + Arrays.toString(keyNames)
-                    + " are present in input table with columns "
-                    + Arrays.toString(input.getDefinition().getColumnNamesArray()) + ". Missing columns: "
-                    + Arrays.toString(missingColumns));
-        }
+        final List<String> keyNames = groupByColumns.stream().map(ColumnName::name).collect(Collectors.toList());
```

Comment:
We were probably better off with an array that can be wrapped as a list in O(1) storage, rather than a list that can only be converted to a list in O(n) storage... not a big deal, but not an improvement.